extensions/adobe/experience/decisioning/proposition-interaction-detail.example.1.json

```diff
@@ -26,6 +26,10 @@
         "xdm:algorithmID": "randomuniform",
         "xdm:trafficType": "noncontextual"
       }
-    ]
+    ],
+    "xdm:characteristics": {
+      "xdm:prop1": "Value 1",
```

Comment:
I believe we should drop the `xdm:` prefix in the examples here to indicate this property bag carries any key-value pairs.

Reply:
ok, thank you